provide-storage/nav.yml

```diff
@@ -0,0 +1,5 @@
+nav:
+ - 'Overview': index.md
+ - 'Quickstart': quickstart.md
```

Comment:
Is there a base branch for the provide storage section? Or is the plan to release as pages are approved? These both don't exist. If we were going to use a base branch, I'd let something like this go. Since right now we're targeting `main`, I'd prefer to not define pages here that don't exist (even if it doesn't impact the live site)